src/ray/common/ray_config_def.h

```diff
@@ -57,6 +57,11 @@ RAY_CONFIG(uint64_t, gcs_pull_resource_loads_period_milliseconds, 1000)
 /// heartbeat intervals, the raylet monitor process will report
 /// it as dead to the db_client table.
 RAY_CONFIG(int64_t, num_heartbeats_timeout, 30)
+
+/// If GCS restarts, before the first heatbeat is sent, initial_num_heartbeats_timeout
```

Comment:
we can add a bit detail why this has to be different from the default timeout value.